specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/disk.json

```diff
@@ -820,6 +829,15 @@
           "type": "string",
           "description": "ARM id of the DiskAccess resource for using private endpoints on disks."
         },
+        "burstingEnabledTime": {
```

Comment:
This property cannot be updated by the client. Should not be in DiskUpdateProperties

Reply:
Remove from Disk Update